docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-configuration.md

```diff
@@ -321,7 +321,7 @@ worker-node-3  65.21.4.103:10124    Online  zone=eu-east-1
 worker-node-3  37.27.13.10:10124    Online  zone=us-west-1
 ```
 
-In this case, the replica of the volume will be placed on any nodes i.e.
+In this case, the volume replicas will be provisioned on any nodes i.e.
```

Comment:
"on any nodes" seems incorrect, if you look at the context.

Reply:
`on the below given nodes` seems correct.